Atos20/Crate
# Nd style survey categories

What’s this PR do?  
- Updates the StyleSurvey to have more than images.
- Changes the code that is used to access the images to display them on the page.
- Allows a user to navigate backward and forward through the quiz.
- Clicking on an ImageTile image will collect that answer into userAnswers state.
- Fixes the tabbing to 2 spaces to fit the style of all files in the project.
 
Where should the reviewer start?  
- Git pull this branch
- Run `npm i` to install all dependencies that may be missing inside of the web directory
 
How should this be manually tested?  
- Navigate to the main api directory (outside web) and `npm start` to open DB connections.
- Navigate to the web directory and `npm start` -> navigate to localhost:3000.
- Login with a user
- Navigate to Crates path from navbar.
- Click subscribe on any of the crates to launch the survey.
- Click back and forward buttons to ensure the category name changes (pictures will remain the same).
- Click on any of the pictures to log that answer, check local state to ensure they are increasing properly.
- Click on the submit button at the very end to log the answers to console.
 
Any background context you want to provide?  
- Additional styling to fit wire frame will be done in a future branch.
- Finalizing functionality will be done in a future branch.
 
What are the relevant tickets?  
- fix #10 
- fix #30
- fix #31
 
Screenshots (if appropriate)  
n/a
 
Questions: 
n/a
